### PR TITLE
add easyconfig for cuDNN 4.0 and 5.0

### PR DESCRIPTION
**Pull request by thiell**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/c/cuDNN/cuDNN-5.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+homepage = 'https://developer.nvidia.com/cudnn'
+description = """The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for
+    deep neural networks."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# Nvidia developer registration required.
+# Download link: https://developer.nvidia.com/rdp/cudnn-download
+sources = ['%(namelower)s-7.5-linux-x64-v%(version)s-rc.tgz']
```
Comment (boegel) [inline easybuild/easyconfigs/c/cuDNN/cuDNN-5.0.eb]: should the version be `5.0-rc` here, since this is a release candidate?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ec23f29a65129310625983742ef02f45 for a full test report.

**Comment by boegel** [comment]
@thiell Looks good, other than the cuDNN 5.0 version which should probably be 5.0-rc?

File: easybuild/easyconfigs/c/cuDNN/cuDNN-5.0.eb (hunk 1 above)
Comment (thiell) [inline easybuild/easyconfigs/c/cuDNN/cuDNN-5.0.eb]: done, thanks @boegel. Actually I added 5.0 as I saw it on the web site but didn't notice it was a RC. I just installed it on our production system and it seems to be OK (see my latest commit).

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
gligar02.gligar.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/90edeba052af335a347ed6ec9cc13495 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @thiell!

**Pull request merged**